routes/web.php

```diff
@@ -18,3 +18,8 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+//temporary
+Route::get('/dashboard', function () {
+    return view('dashboard');
```

Comment:
will create a controller for the dashboard?

Reply:
It doesn't exit even in latest working branch. If I should make it, I'll make.

Reply:
Yes I will. Then I don't have to do that in this PR?